site/content/docs/multistep-checks/_index.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Getting started
+weight: 14
+slug: /
+menu:
+  resources:
+    parent: "Multistep API checks"
+    identifier: "multistep-checks"
+aliases:
+    - /docs/multistep-checks/quickstart/
+    - /docs/multistep-checks/getting-started/
+cli: true
+beta: true
+---
+
+This guide gives you all the info to create your first Multistep API check with Checkly. You should have some prior
+knowledge of working with JavaScript and/or Node.js.
+
+## What is a Multistep API check?
+
+A Multistep API check allows you to write Node.js scripts that can run multiple API requests in sequence, with arbitrary code between requests. This allows you to monitor entire API user flows with a single check, working with response data in between requests exactly as a user of your API would to ensure that these interactions result in the correct results.
+
+Examples of API sequences might be:
+* Users can authenticate and access restricted data
+* Users can get, set and remove data from their account
+* Users can add items to a shopping cart, checkout and go through the payment flow.
+
+Monitoring your API user flows instead of individual endpoints gives confidence that your product as a whole works as intended and that the expected interactions between API calls are functional.
+
+Multistep API checks are powered by [Playwright Test](https://playwright.dev/docs/api-testing), a robust open-souce test-runner.
```

Comment:
I would modify this slightly just to make it super clear that a Multistep check *is* a playwright/test based test, however, you won't have `browser`/`context`/etc available because its using their different *API Testing* setup. Maybe somethign along the lines of:

> Multistep API checks are powered by @playwright/tests's [API Testing](https://playwright.dev/docs/api-testing) mode. Meaning you get all of the power of our typical API tests, in the form of a programmable @playwright/test check

Reply:
Updated, good point 👍